html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+
+  <h1>Coding Form!</h1>
+
+  <label for="email">Email<br></label>
```

Comment:
avoid the use of `br` tags. if you want to break an input onto a new line, wrap it with a div